cmd/podman/spec.go

```diff
@@ -550,6 +551,25 @@ func (c *createConfig) GetVolumeMounts() ([]spec.Mount, error) {
 			Options:     options,
 		})
 	}
+
+	// volumes from image config
+	for _, vol := range c.ImageVolumes {
+		mount := spec.Mount{
+			Destination: vol,
+		}
+		if c.ImageVolumeType == "bind" {
```

Comment:
I would prefer that we handle this in libpod as opposed to sending down mounts with no source

Reply:
fixed